examples/cryptocurrency-advanced/frontend/package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Exonum Team <exonum@bitfury.com>",
   "license": "Apache-2.0",
   "scripts": {
-    "proto": "pbjs --keep-case -t static-module ../../../exonum/src/proto/schema/exonum/protocol.proto ../backend/src/proto/cryptocurrency.proto -o ./proto/stubs.js",
+    "proto": "pbjs --keep-case -t static-module node_modules/exonum-client/proto/**/*.proto ../backend/src/proto/service.proto -o ./proto/stubs.js",
```

Comment:
I *think* you should use `--path` for includes (that is, `--path node_modules/exonum-client/proto`), at least this would be similar to `protoc` workflow. Can you try this approach?

Reply:
With `--path` it does not work as expected :(